src/main/views/accessibility-statement.njk

```diff
@@ -0,0 +1,79 @@
+  </ul>
+  <p class="govuk-body-m">We’ve also made the website text as simple as possible to understand.</p>
+  <p class="govuk-body-m"><a class="govuk-link" href="https://mcmw.abilitynet.org.uk/">AbilityNet</a> has advice on making your device easier to use if you have a disability.</p>
+
+  <h1 class="govuk-heading-xl">How accessible this website is</h1>
+  
+  <p class="govuk-body-m">This website is fully compliant with the <a class="govuk-link" href="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines version 2.1</a> AA standard.</p>
+  <p class="govuk-body-m">There are no known accessibility issues within this service.</p>
+  <p class="govuk-body-m">Feedback and contact information</p>
+  <p class="govuk-body-m">If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille, you can:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>Email: HMCTSforms@justice.gov.uk</li>
+    <li>Call: +44 (0) 300 123 1711</li>
+    <li>Contact your service representative</li>
+  </ul>
+  <p class="govuk-body-m">We’ll consider your request and get back to you in 10 working days.</p>
+
+  <h1 class="govuk-heading-xl">Reporting accessibility problems with this website</h1>
+  <p class="govuk-body-m">We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>Email: customer.service@justice.gov.uk</li>
+    <li>Call: +44 (0) 300 123 1711</li>
+    <li>Enforcement procedure</li>
+  </ul>
+  <p class="govuk-body-m">The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’).</p>
+  <p class="govuk-body-m">If you’re not happy with how we respond to your complaint, <a class="govuk-link" href="https://www.equalityadvisoryservice.com/">contact the Equality Advisory and Support Service (EASS).</a></p>
+
+  <h1 class="govuk-heading-xl">Contacting us by phone or visiting us in person</h1>
+  <p class="govuk-body-m">We provide a text relay service for people who are D/deaf, hearing impaired or have a speech impediment.</p>
+  <p class="govuk-body-m">Our offices and tribunal venues have audio induction loops, or if you contact us before your visit we can arrange a British Sign Language (BSL) interpreter. You can also request step-free access or a foreign language interpreter.</p>
+  <p class="govuk-body-m">If you have a question about accessibility in our Tribunal venues, you can contact the First-tier Tribunal enquiry line: +44 (0) 300 123 1711. You can find the contact details for court and tribunal venues.</p>
+  <p class="govuk-body-m">You can also contact your service representative for more information.</p>
```

Comment:
In the attachment document from the ticket, this line "find the contact details for court and tribunal venues" has a link. This is missing here. Can you add that please?